screens/rewards/RewardsScreen.js

```diff
@@ -73,6 +82,42 @@ export default class RewardsScreen extends React.Component {
       return null;
     }
 
+    if (this.state.isGuest) {
+      return (
+        <View style={{ flex: 1 }}>
+          <NavHeaderContainer vertical backgroundColor={Colors.primaryGreen}>
+            <NavButton onPress={() => this.props.navigation.goBack()}>
+              <FontAwesome5 name="arrow-down" solid size={24} color="white" />
+            </NavButton>
+            <BigTitle
+              style={{
+                marginLeft: 18,
+                color: Colors.lightest,
+                fontSize: 36,
+              }}>
+              How it Works
+            </BigTitle>
+          </NavHeaderContainer>
+          <Image
```

Comment:
Need an extra `ScrollView` around this image and button. Otherwise, I can't see the button on iPhone 8
![image](https://user-images.githubusercontent.com/21160510/78525491-264e1680-778c-11ea-84c7-9d644cb3ee5a.png)

Reply:
On a side note, does the graphic look a bit pixelated to you? This also applies to the loading screen on my phone (in another branch)

Reply:
oh yeah it's def blurry, if anyone has a higher res vers pls send